logbat/src/main/java/info/logbat/domain/log/presentation/LogController.java

```diff
@@ -23,16 +23,15 @@ public class LogController {
 
   @PostMapping
```

Comment:
이렇게 응답 status를 정하는건 어떤가요?
해당 API는 성공했을때 201을 보내야 하니 응답 코드에 대한 명시가 함수 내부에 있는것 보다 함수 정의 바로 위에 있는면이 개인적으로 더 보기 좋다고 생각해서 제안드립니다!

```suggestion
  @PostMapping
  @ResponseStatus(HttpStatus.CREATED)
```

Reply:
이렇게 하면 함수 리턴타입을 void로 해야할 것 같아요